src/dotnet-scaffolding/dotnet-scaffold-aspnet/Templates/BlazorCrud/Edit.cs

```diff
@@ -165,7 +165,7 @@ public virtual string TransformText()
             this.Write(this.ToStringHelper.ToStringWithCulture(primaryKeyName));
             this.Write(" == ");
             this.Write(this.ToStringHelper.ToStringWithCulture(primaryKeyNameLowerInv));
-            this.Write(");\r\n    }\r\n}\r\n");
+            this.Write(");\r\n    }\r\n}\r\nF\r\n");
```

Comment:
Typo ...

```suggestion
            this.Write(");\r\n    }\r\n}\r\n}\r\n");
```

Reply:
It is autogenerated content as part of this templating engine.  I'll double check why the `F` string is present, could be a typo in the template or just a carry over from a previous string write. We can control the generated content but it would have to be edited everytime this template is change so unless the content (.razor/.cs etc) written to file is incorrect, I kinda leave it be.

Reply:
yep in this case there an extra `F` at the end of the Edit.tt file, fixed. tyty